### PR TITLE
feat(insulter): create new insulter in favor of offline insultgenerat…

### PR DESCRIPTION
**Pull request by JonasEtzold**
…or.org

### CODE REVIEW

File: insult.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        {insults:"[NAME] is a total loser."},
+        {insults:"Losers such as [NAME] use me for publicity for themselves. They are strictly third rate."},
+        {insults:"[NAME] is a totally overrated clown who speaks without knowing the facts. "},
+        {insults:"[NAME] is truly as dumb as rock."},
+        {insults:"[NAME] is a dummy. Just look at [NAME]'s past."},
+        {insults:"[NAME] works really hard but is a guy who just doesn't have it--a total loser! "}
+    ];
+
+    if (!name) {
+        name = "No sentence writer"
```
Comment (bpoetzschke) [inline insult.js:50]: Shouldn't it just return an error in case there is no name set?

File: insult.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    insult = trumpInsults[getRandomInt(0, trumpInsults.length-1)].replace(/[NAME]/g, name); //Replace "[NAME]".
+
+    return callApi('random', `tag=${q}`)
```
Comment (bpoetzschke) [inline insult.js]: What does this do?

File: index.js [hunk 3]
```diff
@@ -173,20 +173,20 @@ controller.on('ambient', function ambient(bot, message) {
...
-                } else {
-                    replyText += `that's not a sentence: ${insult}`;
-                }
-                console.log(replyText);
-                bot.reply(message, {
-                    username: 'bingo',
-                    text: replyText,
-                    icon_emoji: `:anton:`
-                })
+            let insulttext = insult.generate(message.user);
```
Comment (bpoetzschke) [inline index.js:176]: You need to wrap `message.user` either here or in the `insult.js` into `<@${...}>`. Or is this don here: https://github.com/bpoetzschke/slack-bingo-bot/pull/4/files#diff-99644a4767c92b6b164730a5d91919e5R55

File: insult.js (hunk 1 above)
Comment (JonasEtzold) [reply insult.js:50]: I guess falling back to some default is better in this case.

File: insult.js (hunk 2 above)
Comment (JonasEtzold) [reply insult.js]: Stupid me...will correct that.

File: index.js (hunk 3 above)
Comment (JonasEtzold) [reply index.js:176]: Yepp, the second thing is right.

**Comment by JonasEtzold** [comment]
Please review again @bpoetzschke.

**Review by bpoetzschke — APPROVED** [review]

**Pull request merged**